shard/src/ext/settings_cog.py

```diff
@@ -342,6 +406,11 @@ async def roleids(self, ctx):
             lem.add(role.name, role.id)
         await ctx.channel.send(embed=lem.get_embed())
 
+    @commands.command(aliases=['listsettingstags', 'settingstags', 'stags'])
+    async def list_settings_tags(self, ctx):
+        '''Get the list of tags for the settings command'''
+        await ctx.channel.send(f"Tags for settings are: {self.list_tags()}")
```

Comment:
not sure if this command is necessary...

Reply:
I like it, although the default name is a little clumsy